### ISSUE TITLE
bin/kshell.sh fails with java.lang.NoClassDefFoundError: com/github/khud/kshell/repl/UtilKt

### ISSUE THREAD

**Issue by holgerbrandl** [khud/sparklin#30]
To reproduce run
```bash
docker pull maven:3.5.4-jdk-8

docker run --rm -it maven:3.5.4-jdk-8 /bin/bash --login

git clone https://github.com/khud/sparklin
cd sparklin
mvn clean install

bin/kshell.sh
```
which fails with
```
Exception in thread "main" java.lang.NoClassDefFoundError: com/github/khud/kshell/repl/UtilKt
	at com.github.khud.sparklin.kshell.KotlinShell.main(KotlinShell.kt:21)
Caused by: java.lang.ClassNotFoundException: com.github.khud.kshell.repl.UtilKt
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:349)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 1 more
```

**Comment by khud** [comment]
Sorry for that @holgerbrandl, I will promote `kshell-repl-api-0.2.5-1.2.60` to Maven Central tomorrow morning and will be more careful with master branch.

**Comment by khud** [comment]
I released `kshell-repl-api-0.2.5-1.2.60`. Need some time to sync with Maven Central.
